File: javascript/문자열 묶기.js
const solution = (strArr) => {
    const dict = {};
    strArr.forEach(item => {
        const len = item.length;
        if(dict[len] == null && dict[len] == undefined) {
            dict[len] = [];
        }
        dict[len].push(item);
    });
    const values = Object.values(dict).map(a => a.length);
    return Math.max(...values);
}

File: java/문자열 묶기.java
import java.util.Arrays;

class Solution {
    public int solution(String[] strArr) {
        int[] count = new int[strArr.length];
        for(int i = 0 ; i < strArr.length; i++) {
            count[strArr[i].length()]++;
        }
        return Arrays.stream(count).max().getAsInt();
    }
}